ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController.swift

```diff
@@ -496,6 +527,25 @@ class AvatarGroupDemoController: DemoTableViewController {
         }
     }
 
+    private var isUsingImageBasedCustomColor: Bool = false {
+        didSet {
+            updateAvatarsCustomColorRing(index1: 0, index2: avatarCount - 1)
+        }
+    }
+
+    private func updateAvatarsCustomColorRing(index1: Int, index2: Int) {
```

Comment:
Can you update this to something like `updateAvatarsCustomColorRing(for range: Range<Int>)` and call it something like `updateAvatarsCustomColorRing(for: index1..<index2)`? I think this would be cleaner if we used ranges and the half open range operator.